pennylane/qchem/openfermion_obs.py

```diff
@@ -812,19 +812,23 @@ def molecular_hamiltonian(
     charge=0,
     mult=1,
     basis="sto-3g",
-    package="pyscf",
+    method="pyscf",
```

Comment:
Do you think that we default the method to `dhf`? 🤔

Reply:
It is the default.